### PR TITLE
Adding Oracle Linux Cloud Native Environment to the list of adopters.

### PR DESCRIPTION
**Pull request by Djelibeybi**
Signed-off-by: Avi Miller <avi.miller@oracle.com>

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Adds [Oracle Linux Cloud Native Environment](https://www.oracle.com/it-infrastructure/software.html) to the `ADOPTERS.md` file.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 Below is a non-exhaustive list of CRI-O adopters in production environments:
 
 * [Red Hat's Openshift Container Platform](https://www.openshift.com/) uses CRI-O as the only supported CRI implementation starting with the release of OpenShift 4. CRI-O was chosen because it provides a lean, stable, simple and complete container runtime that moves in lock-step with Kubernetes, while also simplifying the support and configuration of clusters.
+
+* [Oracle Linux Cloud Native Environment](https://www.oracle.com/it-infrastructure/software.html) has used
```
Comment (haircommander) [inline ADOPTERS.md:7]: could you remove the newline please? It should correctly wrap to be fitted to screen

**Comment by haircommander** [comment]
once nit, otherwise LGTM
/ok-to-test
/approve
thanks @Djelibeybi

File: ADOPTERS.md (hunk 1 above)
Comment (Djelibeybi) [reply ADOPTERS.md]: I'm so used to hard wrapping Markdown files, I think this is muscle memory. Removed.

Comment (haircommander) [reply ADOPTERS.md]: thanks! could I bother you to squash the commits together as well?

Comment (Djelibeybi) [reply ADOPTERS.md]: Squashified.

**Review by saschagrunert — APPROVED** [review]
LGTM

File: ADOPTERS.md (hunk 1 above)
Comment (haircommander) [reply ADOPTERS.md]: perfection :sunglasses:

**Comment by haircommander** [comment]
LGTM

**Comment by haircommander** [comment]
@umohnani8 @mrunalp @TomSweeneyRedHat PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm

**Review by mrunalp — APPROVED** [review]

**Comment by umohnani8** [comment]
LGTM
/retest

**Comment by haircommander** [comment]
/retest

it's always the most innocent of PRs

**Pull request merged**